app/src/main/java/com/aboolean/movies/domain/usecase/GetPopularMoviesUseCase.kt

```diff
@@ -7,6 +7,12 @@ import io.reactivex.Maybe
 
 interface GetPopularMoviesUseCase : BaseUseCase {
     fun getPopular(page: Int): Maybe<List<Movie>>
+    suspend fun suspendGetPopular(page : Int) : List<Movie>{
+        TODO("You must implement this method if you want to use coroutines")
```

Comment:
I think this line is not necessary because it's already implemented, what do you think?

Reply:
Yeah! I forgot to remove that line hehe. I am going to clean these interfaces.